### PR TITLE
Document installation on macOS

### PR DESCRIPTION
**Pull request by dominiklohmann**
When installing via Homebrew, this is the resulting launchd configuration.

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
  <key>Label</key>
  <string>homebrew.mxcl.vast</string>
  <key>ProgramArguments</key>
  <array>
    <string>/usr/local/opt/vast/bin/vast</string>
    <string>start</string>
  </array>
  <key>ProcessType</key>
  <string>Interactive</string>
  <key>RunAtLoad</key>
  <true/>
  <key>WorkingDirectory</key>
  <string>/usr/local/var/vast</string>
  <key>StandardOutPath</key>
  <string>/usr/local/var/log/vast.log</string>
  <key>StandardErrorPath</key>
  <string>/usr/local/var/log/vast.log</string>
</dict>
</plist>
```

### CODE REVIEW

**Comment by dominiklohmann** [comment]
I've integrated the feedback. Note that this is still blocked by tenzir/homebrew-tenzir#1.

**Comment by mavam** [comment]
The last commit seems to have screwed something up...

**Comment by dominiklohmann** [comment]
> The last commit seems to have screwed something up...

Indeed, that were too many backticks. Fixed, and marked ready for review now that the formula exists.

File: INSTALLATION.md [hunk 1]
```diff
@@ -57,7 +57,22 @@ TODO
 
 ### macOS
 
-TODO
+A current version of VAST can be installed using Homebrew.
+
+Install VAST from tenzir/tenzir tap (current master):
```
Comment (0ortmann) [inline INSTALLATION.md]: It's a matter of taste, but I'd make it more clear what gets installed and put the install target first:
```suggestion
Install the *current master* from tenzir/tenzir tap:
```

File: INSTALLATION.md [hunk 2]
```diff
@@ -57,7 +57,22 @@ TODO
...
+```sh
+brew install --HEAD tenzir/tenzir/vast
+```
+
+Install VAST from tenzir/tenzir tap (current release):
```
Comment (0ortmann) [inline INSTALLATION.md]:
```suggestion
Install the *latest release* from tenzir/tenzir tap:
```

**Pull request merged**